cli/pkg/providers/minikube/minikube.go

```diff
@@ -45,114 +43,49 @@ func (p *provider) GetOverrides() template.Variables {
 }
 
 func (p *provider) CheckEnvironment() error {
-	return workspace.CheckBinaries([]string{"minikube"})
-}
-func (p *provider) PreInstallation(workspace string) (string, error) {
-	return workspace, nil
+	return workspace.CheckBinaries([]string{"minikube", "terraform"})
 }
-
-func (p *provider) Provision(providerConfig map[string]string, dir workspace.ConfigDir) (kube.KubeCtx, error) {
-	if err := p.startCluster(providerConfig); err != nil {
-		return kube.KubeCtx{}, err
+func (p *provider) PreInstallation(workspacePath string) (string, error) {
+	remoteUrl := "github.com/airyhq/airy/infrastructure/terraform/install"
+	installDir := workspacePath + "/terraform"
+	installFlags := strings.Join([]string{"PROVIDER=minikube", "WORKSPACE=" + workspacePath}, "\n")
+
+	var gitGetter = &getter.Client{
```

Comment:
Now that we have two providers, and are soon to support more, i.e `aws-ec2` and GCP, the way we have it set up, it will download all files in the `install` directory including the other provider directories. Do we want this? 

Perhaps a fix would be to do two separate calls? One where we append "/core" to `remoteUrl` in line 54, and then do something like `gitGetter.Src = remote URL + "provider_name". I'm sure we could also do them using go routines too.

Reply:
Good point :+1:. I'd open an issue for this for now.